requirements_gpu.txt

```diff
@@ -3,5 +3,5 @@
 
 # Torch GPU Version
 --find-links https://download.pytorch.org/whl/torch_stable.html
-torch==1.5.0
-torchvision==0.6.0
\ No newline at end of file
+torch==1.8.0
+torchvision==0.9.0
```

Comment:
So what happens with `torch` is that I tested on Ubuntu 20 with 3090 and actually ran into issues where 1.8.0.+rocm4.0.1 got installed.